### DISCUSSION TITLE
Automatic update

### DISCUSSION THREAD

**Discussion by Uilsonjw** [sircharlo/meeting-media-manager#349] [Ideas] [answered]
It's very nice that the MMM has the ability of the automatic update, however could be that in a specific situation for a specific reason we would like to postpone the update.

So my sugestion is that stead the software updates while it is opened, that the software shows a message that a new version is avaliable and that the recomendatios is that the software is updated. So the user can decide update it in this time or postpone it for late. Ofcourse, the message that the update is avaliable should appear every time that the MMM is opened until  it is updated.

Thanks

**Comment by sircharlo** [comment, marked as answer]
Already implemented [here](https://github.com/sircharlo/meeting-media-manager/commit/213cb7ff8613cf125cf6edbb10c44104e2b55711) and waiting for release of next version.

**Comment by Uilsonjw** [comment]
Very nice. The next version will come with a lot changes and new resources. Thanks a lot.

**Reply by sircharlo** [reply]
[Try it out! ](https://github.com/sircharlo/meeting-media-manager/releases/latest)
